pynecone/components/media/icon.py

```diff
@@ -13,3 +13,24 @@ class Icon(ChakraIconComponent):
     """An image icon."""
 
     tag = "None"
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Let's move this logic to the `create` method, similar to how we do for components like [slider](https://github.com/pynecone-io/pynecone/blob/d0304659b8fc6dd4abf250ebea55d7aa5e0e8d19/pynecone/components/forms/slider.py#L4)